oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -479,9 +479,9 @@ Notes
        can be seen in much literature that starts from the assumption
        that maintainers are essential and instead asks, what makes a
        maintainer great, what are the best-practices maintainers
-       should follow, how do we ensure maintainer don't burn out, how
+       should follow, how do we ensure maintainers don't burn out, how
        do we ensure critical parts of the software supply chain, like
-       OpenSSL are maintained, etc.  I have included some relevant
+       OpenSSL is maintained, etc.  I have included some relevant
```

Comment:
Rervert this change ("are" is referring to the plural "critical parts")

Reply:
You're absolutely right!